apps/stripe/src/app/api/saleor/transaction-cancelation-requested/use-case-response.ts

```diff
@@ -12,12 +12,19 @@ import {
 class Success extends SuccessWebhookResponse {
   readonly transactionResult: CancelSuccessResult;
   readonly saleorMoney: SaleorMoney;
+  readonly date: Date | null;
```

Comment:
maybe timestamp is a better name?

Reply:
Done in https://github.com/saleor/apps/pull/1918/commits/9ccceda22062668a97815425fc589ca00e585931